### PR TITLE
Tokens - Adds New Rebrand Colors and Font

### PR DESCRIPTION
**Pull request by anechol**
# Description

Adds in new color and font tokens for the rebrand. Does not include updating any components with new values.

[DSS-667](https://kajabi.atlassian.net/browse/DSS-667)

## Type of change

- [x] Token change (adds, updates, or removes design tokens)

# How Has This Been Tested?

- [ ] unit tests
- [ ] e2e tests
- [ ] accessibility tests
- [x] tested manually
- [ ] other:

[DSS-667]: https://kajabi.atlassian.net/browse/DSS-667?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ju-Skinner — APPROVED** [review]
LGTM

**Comment by ju-Skinner** [comment]
I have some additional questions about the number scale. I'll circle back after our meeting later today.

**Comment by anechol** [comment]
@ju-Skinner Tokens have been updated

**Review by ju-Skinner — APPROVED** [review]

**Review by monicawheeler — CHANGES_REQUESTED** [review]
Sounds like I am +1'ing Julian but to be sure, the color scale has appeared to be changed to a 10s scale
image

**Comment by anechol** [comment]
> Sounds like I am +1'ing Julian but to be sure, the color scale has appeared to be changed to a 10s scale image

@monicawheeler Per team discussion, we've decided to move forward with the 100 scale tokens.

File: libs/core/src/global/styles/tokens/core/_core.scss [hunk 1]
```diff
@@ -21,47 +21,91 @@
...
-  --pine-color-orange-300: #fb7e09;
-  --pine-color-orange-400: #b3501e;
-  --pine-color-orange-500: #5a260c;
-  --pine-color-purple-100: #eee4f6;
-  --pine-color-purple-200: #d9c2ef;
-  --pine-color-purple-300: #8e5ad8;
-  --pine-color-purple-400: #50348a;
-  --pine-color-purple-500: #381c5e;
-  --pine-font-family-circular: "Circular", -apple-system, "BlinkMacSystemFont", '"Segoe UI"', "Roboto", '"Helvetica Neue"', "Arial", '"Noto Sans"', sans-serif;
+  --pine-color-gray-50: #fcfcfc;
```
Comment (pixelflips) [inline libs/core/src/global/styles/tokens/core/_core.scss]: after today's critique meeting, we've agreed to stay with the `grey` naming for now.

**Review by pixelflips — APPROVED** [review]
One minor comment, otherwise LGTM! Nice work!

**Pull request merged**